### PR TITLE
✨ feat(dashboard): radio field component

### PR DESCRIPTION
**Pull request by emileNetter**
Capture d’écran 2021-04-21 à 18 07 26
Capture d’écran 2021-04-21 à 18 07 11

### CODE REVIEW

File: packages/junipero/lib/RadioField/index.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    dispatch({ checked: checked ?? false });
+  }, [checked]);
+
+  useEventListener('keypress', e => {
+    onKeyPress_(e);
+  }, globalEventsTarget);
+
+  useImperativeHandle(ref, () => ({
+    innerRef,
+    inputRef,
```
Comment (dackmin) [inline packages/junipero/lib/RadioField/index.js:43]: missing `internalValue: state.checked, focused: state.focused`

File: packages/junipero/lib/RadioField/index.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  }));
+
+  const onKeyPress_ = e => {
+
```
Comment (dackmin) [inline packages/junipero/lib/RadioField/index.js:47]: remove useless linebreak

File: packages/junipero/lib/RadioField/index.stories.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        description="This is a description"
+        onChange={value => onChange(value, 'strong')}
+        value='strong'
+        checked={intelligence === 'strong'}
+      />
+      <RadioField
+        label="This one's dumb"
+        description="This is a description"
+        onChange={value => onChange(value, 'strong')}
+        value='strong'
```
Comment (dackmin) [inline packages/junipero/lib/RadioField/index.stories.js:139]: dumb

File: packages/junipero/lib/RadioField/index.styl [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    
+    .label
+      margin-left: 20px
+
+      & :first-child
+        line-height: $text-default-line-height
+
+      .description
+        font-size: $text-extra-font-size 
+        line-height: $text-extra-line-height
```
Comment (dackmin) [inline packages/junipero/lib/RadioField/index.styl:65]: missing EOF line break

File: packages/junipero/lib/ToggleField/index.js [hunk 5]
```diff
@@ -49,6 +49,7 @@ const ToggleField = forwardRef(({
       state.checked = !state.checked;
       dispatch({ checked: state.checked });
       e.preventDefault?.();
+      onChange({ value, checked: state.checked });
```
Comment (dackmin) [inline packages/junipero/lib/ToggleField/index.js:52]: should not be here

**Review by dackmin — CHANGES_REQUESTED** [review]

**Comment by dackmin** [comment]
In addition to my review, this is a breaking change from v1 where you would have supplied an `options` prop to render multiple radio fields. Is there any particular reason you didn't start from v1 RadioField?
@NicolasAuger whatcha think

**Review by NicolasAuger — CHANGES_REQUESTED** [review]

**Comment by NicolasAuger** [comment]
The component is nice, i prefer using `opts` for multiple fields too, rather than multiple `<RadioField />` components.

**Review by dackmin — CHANGES_REQUESTED** [review]

**Review by dackmin — APPROVED** [review]

**Pull request merged**